docker/dev/php/Dockerfile

```diff
@@ -2,43 +2,52 @@ FROM php:7.3-cli-alpine3.10
 
 # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
 ENV COMPOSER_ALLOW_SUPERUSER 1
-ARG HOST_USER
 ARG HOST_USER_ID
 
-COPY files/ini /phpIni
+COPY files/php /phpIni
```

Comment:
you can actually remove this line since you  replaced xdebug with pcov :smile: 
just create an empty /phpIni dir